packages/logo/logo.scss

```diff
@@ -0,0 +1,50 @@
+@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+
+$jkl-logo-color: $svart;
+$jkl-logo-color--negative: $snøhvit;
+$jkl-logo-magic-origin--f: 30px 132px; /* Is connected to the viewbox of the logo, change at your own risk */
+$jkl-logo-magic-origin--line: 34.3px; /* Must be the same as the line x1 origin, change at your own risk */
+
+.jkl-logo {
+    &--animated {
```

Comment:
I think the current animation feels a bit weird. There are some weird bouncing going on and also the animation is quite different from the example on brandspace.

Reply:
We have to look at core animation soon regardless, so this is a great place to look for inspiration for the more expressive animations. I think some "deep" s-curves, `ease-in-out`-style could look good here on most transitions. Maybe even some linears 😮

Reply:
Yeah, let's find some good standardized curves to work with, there is too much custom stuff already

Reply:
Reverted to more basic animation curves. We should still standarize these. Do that in a own PR.